tests/stubtest_third_party.py

```diff
@@ -128,6 +137,80 @@ def run_stubtest(dist: Path, *, verbose: bool = False, specified_platforms_only:
     return True
 
 
+# Performs some black magic in order to run stubtest inside uWSGI
+# we have to write the exit code from stubtest to a surrogate file
+# because uwsgi --pyrun does not exit with the exitcode from the
+# python script. We have a second wrapper script that passed the
+# arguments along to the uWSGI script and retrieves the exit code
+# from the file, so it behaves like running stubtest normally would
+def setup_uwsgi_stubtest_command(dist: Path, venv_dir: Path, stubtest_cmd: list[str]) -> bool:
```

Comment:
Multiline comment -> docstring

```suggestion
def setup_uwsgi_stubtest_command(dist: Path, venv_dir: Path, stubtest_cmd: list[str]) -> bool:
    """Perform some black magic in order to run stubtest inside uWSGI.

    We have to write the exit code from stubtest to a surrogate file
    because uwsgi --pyrun does not exit with the exitcode from the
    python script. We have a second wrapper script that passed the
    arguments along to the uWSGI script and retrieves the exit code
    from the file, so it behaves like running stubtest normally would.
    """
```